airgun/widgets.py

```diff
@@ -333,8 +333,8 @@ def read(self):
 
 class Search(Widget):
     search_field = TextInput(locator=(
-        ".//input[@id='search' or @ng-model='table.searchTerm' or "
-        "contains(@ng-model, 'Filter')]"))
+        ".//input[@id='search' or @type='search' or "
```

Comment:
Unfortunately, just found out `@type='search'` is not unique (at least on content view details page there're few such elements). Could you please try `@placeholder='Filter...'` instead?